plugins/mortar-pestle-bonk

```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/Erishion-Games-LLC/Mortar_Pestle_Runelite_Plugin.git
+commit=a23f11010dae0935b06ec3e1828920ad078cbcad
```

Comment:
I recommend adding an `authors` field to get rid of the bot's "non-author plugin change" label.
```suggestion
commit=a23f11010dae0935b06ec3e1828920ad078cbcad
authors=Ben-Colwell
```